### PR TITLE
New post: DriveBackupV2

### PR DESCRIPTION
**Pull request by Mocab**

### CODE REVIEW

File: _posts/minecraft/plugins/backup.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+permalink: /minecraft/plugins/drivebackup/
+---
+
+<div class="install-plugin">
+    <img style="border-radius: 7px;" src="../../../assets/images/posts/plugins/backup/logo.png">
+    <p>DriveBackupV2</p>
+    <a href="https://dev.bukkit.org/projects/drivebackupv2/files/latest">Download this Plugin</a>
+</div>
+
+# How to automatically backup your server files?
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/backup.md:15]: I feel like this title is useless, as it already is in line 18.

**Comment by itsjuoum** [comment]
I saw that the path for this article's images is `assets/images/posts/plugins/backup/logo.png `. Please consider changing it to `assets/images/posts/plugins/drivebackup/logo.png `, as only "backup" can be confusing and/or used for backup purposes.

File: _posts/minecraft/plugins/backup.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title:  "DriveBackup"
+categories: Minecraft
+tags: plugins
+permalink: /minecraft/plugins/drivebackup/
+---
+
+<div class="install-plugin">
+    <img style="border-radius: 7px;" src="../../../assets/images/posts/plugins/backup/logo.png">
+    <p>DriveBackupV2</p>
+    <a href="https://dev.bukkit.org/projects/drivebackupv2/files/latest">Download this Plugin</a>
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/backup.md:12]: Adding a direct link to download the file isn't a good practice. Please change it to `https://dev.bukkit.org/projects/drivebackupv2/`

File: _posts/minecraft/plugins/backup.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "DriveBackup"
+categories: Minecraft
+tags: plugins
+permalink: /minecraft/plugins/drivebackup/
+---
+
+<div class="install-plugin">
+    <img style="border-radius: 7px;" src="../../../assets/images/posts/plugins/backup/logo.png">
```
Comment (itsjuoum) [inline _posts/minecraft/plugins/backup.md:10]: There's no need for us to store an image if we can get it directly from `https://media.forgecdn.net/avatars/272/319/637250850729446954.png`.

**Review by itsjuoum — CHANGES_REQUESTED** [review]

File: _posts/minecraft/plugins/backup.md (hunk 1 above)
Comment (Mocab) [reply _posts/minecraft/plugins/backup.md:15]: what should i replace it with?

Comment (itsjuoum) [reply _posts/minecraft/plugins/backup.md:15]: As I said, "is useless, as it already is in line 18", so, remove it completely imo

File: _posts/minecraft/plugins/2021-8-10-backup.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+<div class="install-plugin">
+    <img style="border-radius: 7px;" src="https://media.forgecdn.net/avatars/272/319/637250850729446954.png">
+    <p>DriveBackupV2</p>
+    <a href="https://dev.bukkit.org/projects/drivebackupv2/">Download this Plugin</a>
+</div>
+
+
+### What is DriveBackupV2?
+Drivebackup is a plugin that automatically backs up your server to the cloud, or to your own device!
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-8-10-backup.md:17]:
```suggestion
DriveBackupV2 is a plugin that will automatically backup your server to the cloud, or to your own device!
```

File: _posts/minecraft/plugins/2021-8-10-backup.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+## Setting up DriveBackupV2
+
+1. Set up the server, you can use Spigot or any of its forks, we recommend using [Purpur](https://purpur.pl3x.net) or [Paper](https://papermc.io).
+2. Open the file manager.
+3. Find and click on the folder named "plugins".
+4. Download the [DriveBackupV2](https://dev.bukkit.org/projects/drivebackupv2/) and upload it to the folder.
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-8-10-backup.md:24]:
```suggestion
4. Download the [DriveBackupV2](https://dev.bukkit.org/projects/drivebackupv2/) plugin and upload it to the folder.
```

File: _posts/minecraft/plugins/2021-8-10-backup.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+5. Restart the server.
+6. Go back to the "plugins" folder, you should see that a new folder was generated, it is usually named "DriveBackupV2".
+7. Read the [docs](https://github.com/MaxMaeder/DriveBackupV2/wiki) for instructions on how to set it up.
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-8-10-backup.md:27]:
```suggestion
7. Read the [plugin wiki](https://github.com/MaxMaeder/DriveBackupV2/wiki) for instructions on how to set it up.
```

File: _posts/minecraft/plugins/2021-8-10-backup.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+8. once you are done setting up everything, restart the server to load changes.
```
Comment (DaddyChimken) [inline _posts/minecraft/plugins/2021-8-10-backup.md:28]:
```suggestion
8. Once setup is done, restart the server to load your changes.
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

**Pull request merged**